CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# v2.1.1 (2024-04-16)
+
+## OS Changes
+ * Maintain full bottlerocket kernel configs ([#88])
```

Comment:
this should go under a "Build Changes" section

Reply:
We are missing a line that explains the microcode changes, also under "Build Changes"

https://github.com/bottlerocket-os/bottlerocket-kernel-kit/commit/f1b0b4f98a5dfb631e86fcf1a71329c71b51ea37